tests/beacon/_utils/test_hash.py

```diff
@@ -9,3 +14,16 @@ def test_hash():
 
 def test_hash_is_keccak256():
     assert hash_eth2(b'foo') == keccak(b'foo')
+
+
+def test_repeat_hash():
+    output = repeat_hash_eth2(b'helloworld', 5)
+    assert len(output) == 32
```

Comment:
This type of test is well suited for property based testing.  You could write one using hypothesis like this:

```python
from hypothesis import strategies as st, given

@given(value=st.binary(), iterations=st.integers(min_value=1))
def test_repeat_hash_always_produces_32_bytes(value, iterations)
    output = repeat_hash_eth2(value, iterations)
    assert len(output) == 32
```